src/components/Shared/StyledTableCell.tsx

```diff
@@ -0,0 +1,15 @@
+import { styled } from '@mui/material/styles';
```

Comment:
this is deprecated: https://github.com/mui/material-ui/blob/master/packages/mui-material/src/styles/createStyles.js#L8-L9

see below comment for styling table rows.

Reply:
https://mozilla-hub.atlassian.net/browse/FXP-2201